src/app_service/service/chat/service.nim

```diff
@@ -436,7 +440,7 @@ QtObject:
 
   proc clearChatHistory*(self: Service, chatId: string) =
     try:
-      let response = status_chat.deleteMessagesByChatId(chatId)
+      let response = status_chat.clearChatHistory(chatId)
```

Comment:
Here on the other hand, we do want to clear the history and trigger a sync, which is done by using `clearChatHistory` (which as part of this PR now calls the correct RPC API)